corehq/apps/commtrack/static/commtrack/ko/locations.js

```diff
@@ -44,7 +25,7 @@ function LocationSettingsViewModel() {
 
     this.new_loctype = function() {
         var parent_pk = (_.last(settings.loc_types()) || {}).pk;
-        var new_loctype = new LocationTypeModel({parent_type: parent_pk}, this);
+        var new_loctype = new LocationTypeModel({parent_type: parent_pk}, commtrack_enabled);
```

Comment:
> I see now that settings is actually a dict containing one key loc_types. I could add another key to that dict called commtrack_enabled, is that what you're suggesting?

yeah that's what i was suggesting. or you could rename it `location_types` and remove the dict wrapping and then passing both in would be logical